packages/grafana-data/src/themes/createComponents.ts

```diff
@@ -69,8 +69,8 @@ export function createComponents(colors: ThemeColors, shadows: ThemeShadows): Th
       background: input.background,
     },
     tooltip: {
-      background: colors.mode === 'light' ? '#555' : colors.background.secondary,
-      text: colors.mode === 'light' ? '#FFF' : colors.text.primary,
+      background: colors.mode === 'light' || 'fusebit' ? '#555' : colors.background.secondary,
+      text: colors.mode === 'light' || 'fusebit' ? '#FFF' : colors.text.primary,
```

Comment:
Should be unnecessary if colors.background.secondary and colors.text.primary are set in the fusebit color theme.